File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 2/SOAL 5/PRAK205-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
#include <math.h>
int main(){
    int A, B;
    scanf("%d %d", &A, &B);
    int C = sqrt((B * B)-(A*A));
    int k = A + B + C;
    int l = 0.5 * A * C;
    printf("Alas = %dcm\n", C);
    printf("Tinggi = %dcm\n", A);
    printf("Keliling = %dcm\n", k);
    printf("Luas = %dcm^2\n", l);
return 0;
}

File: MODUL 2/SOAL 5/PRAK205-2410817210002-Sebastian_Abe_Santoso.py
total = []
while len(total) < 2:
    baris = input()
    total.extend(map(float, baris.split()))
A, B = total[:2]
C = ((B * B)-(A*A))**0.5
k = A + B + C
l = 0.5 * A * C
print(f"Alas = {C:.0f}cm")
print(f"Tinggi = {A:.0f}cm")
print(f"Keliling = {k:.0f}cm")
print(f"Luas = {l:.0f}cm^2")